src/inventory.cpp

```diff
@@ -255,6 +255,20 @@ std::string ItemStack::getDescription(IItemDefManager *itemdef) const
 	return desc.empty() ? name : desc;
 }
 
+std::string ItemStack::getShortDescription(IItemDefManager *itemdef) const
+{
+	std::string desc = metadata.getString("short_description");
```

Comment:
this doesn't seem to be documented anywhere

Reply:
It's documented in `lua_api.txt`:
https://github.com/minetest/minetest/blob/9cec521975f55452b07aef570b5c096c436b807e/doc/lua_api.txt#L1977

Reply:
Whoops.